src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DrawerDriver.java

```diff
@@ -27,11 +28,15 @@ public void operateTo(int x, int y) {
 		line.setStartCoordinates(this.startX, this.startY);
 		line.setEndCoordinates(x, y);
 
-		drawLine(line);
+		DrawPanelController dpc = DrawerFeature.getDrawerController();
```

Comment:
Constructor injection should be used (using a parameter). This is unnecessary direct coupling with DrawerFeature class.